torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -1994,6 +1995,20 @@ bool removeInplaceOperations(const std::shared_ptr<Graph>& graph) {
       graph, [&](Node* node) { return inplace_ops.count(node->kind()) != 0; });
 }
 
+void printFusionGroupsForDebug(std::shared_ptr<Graph>& graph) {
```

Comment:
No need to re-traverse the graph, I would just add this logic here https://github.com/pytorch/pytorch/blob/a04c88f460d6dc461ae25ef7ddfc2687968d6b58/torch/csrc/jit/codegen/cuda/graph_fuser.cpp#L1543